### ISSUE TITLE
Majority of the site is not useable with a keyboard

### ISSUE THREAD

**Issue by emma-sg** [emma-sg/roleypoly#4]
None of the role toggles can be interacted with via keyboard. This prevents anyone who is unable to or prefers not to use a pointing device from meaningfully interacting with the site.

To fix this, I believe the toggles should be replaced with semantic HTML `input` elements – radio buttons and checkboxes. Some css changes will be necessary to make this work.

It would also be possible to re-implement native interactions with HTML `input`s using the existing HTML, however I believe this would be significantly more work than using dedicated HTML elements.
